session.go

```diff
@@ -211,9 +211,10 @@ type session struct {
 }
 
 var (
-	_ Session      = &session{}
-	_ EarlySession = &session{}
-	_ streamSender = &session{}
+	_                       Session      = &session{}
+	_                       EarlySession = &session{}
+	_                       streamSender = &session{}
+	deadlineSendImmediately              = time.Time{}.Add(42 * time.Second) // any value > time.Time{} and before time.Now() is fine
```

Comment:
Maybe decrease this to a smaller amount (1ms?), in case we're running on a system with a broken clock?

Reply:
I was avoiding round numbers, makes it easier to distinguish it from `time.Time{}` when looking at the printed output. I can reduce it to 42ms though.